lib/src/command/dependency_services.dart

```diff
@@ -734,7 +760,7 @@ Future<List<Object>> _computeUpgradeSet(
         ? SolveType.downgrade
         : SolveType.get,
     cache,
-    Package(pubspec, entrypoint.rootDir),
+    Package(pubspec, entrypoint.rootDir, entrypoint.root.workspaceChildren),
```

Comment:
nit: trailing comma

Reply:
I don't think we need it here.

We have a lint requiring a trailing comma when needed.